### DISCUSSION TITLE
how can i fix this error while building this

### DISCUSSION THREAD

**Discussion by blazingcody0707** [Shiny-Miner/CFRU-expansion#6] [General]
![Screenshot 2024-10-02 134342](https://github.com/user-attachments/assets/c6e33f60-defb-4a8d-b381-e0b433c62e48)

**Comment by mZake** [comment]
Open the file MainEvIv.c, located at src folder. There search for #include "../include/new/Vanilla_Functions.h, change Functions with functions. The problem is the 'F' in uppercase

**Reply by blazingcody0707** [reply]
thank you for helping me build this it built now
